spec/features/api/v2/oauth_applications_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe "OAuth applications" do
```

Comment:
may make sense to change the route of this spec file since it doesn't occur in the api (maybe `spec/features/oauth_applications_spec.rb` ?)